src/time/format/formatter.cr

```diff
@@ -23,6 +23,10 @@ struct Time::Format
       io << time.year / 100
     end
 
+    def full_or_short_year
+      year
```

Comment:
Why an alias?

Reply:
The parser has a different implementation.